frappe/templates/includes/search_box.html

```diff
@@ -16,7 +16,7 @@ <h5 class="item-search-results" style="margin-right: 30px;"></h5>
 frappe.ready(function() {
 	if(frappe.utils.get_url_arg("search")) {
 		var txt = frappe.utils.get_url_arg("search");
-		$(".item-search-results").html("{{ _('Search results for') }}: " + encodeURIComponent(txt));
```

Comment:
this will lead to JS injections. we need to strip all `<>` if possible

try

```
$(".item-search-results").html("{{ _('Search results for') + ': ' + html2txt(frappe.form_dict.search) }}"); 
```

test if this works!